### DISCUSSION TITLE
Is it possible to implement SQLite as Sever database?

### DISCUSSION THREAD

**Discussion by vkotov** [Mimetis/Dotmim.Sync#619] [Q&A] [answered]
We use small SQLite databases on server that should be synchronized with clients. Is it it possible to implement support of SQLite on server side (maybe without some features) or it has some restrictions to do this?

Thank you.

**Comment by Mimetis** [comment, marked as answer]
Hey @vkotov 
No, `SQLite `is not meant to be a server side database.
Even if you try it, the `SQLite` `DMS` provider is missing some required features to be functional

**Reply by umerkle** [reply]
@Mimetis 
What features would be required for SQLite DMS Provider? Or in other words, waht would be broken if we would try to use it as a server?

Dotmim.Sync looks extremely useful to us. We want to use it to sync sqlite to MS SQL, sqlite to HTTP and, as mentioned above, sqlite to sqlite. Mostly to not need customers with local installations to install a "full blown real server". More like kind of a peer to peer solution.
